ui/src/core/xibo-cms.js

```diff
@@ -109,6 +109,14 @@ window.XiboInitialise = function(scope, options) {
     scope = ' ';
   }
 
+  // Search for any grids on the page and render them
+  $(scope + " .XiboGrid").each(function() {
+    // Custom Date Range Filter
+    $(this).find('select.XiboDateRangeFilter').on('change', function() {
```

Comment:
With the new refactor for all the JS files, EsLint complains with "Disallow use of this in contexts where the value of this is undefined" in this case (https://eslint.org/docs/latest/rules/no-invalid-this).

Can you please use something like:

```
$(scope + " .XiboGrid").each(function(_idx, el) {
  // Custom Date Range Filter
  $(el).find('select.XiboDateRangeFilter').on('change', function() {
  (...)
```

Reply:
Thank you, Mauro! Please see changes [here](https://github.com/xibosignage/xibo-cms/pull/2827/commits/b972050eb3cdc138c24aeb1e1e98b4b6f6df41b0) :)